sync.py

```diff
@@ -154,19 +155,20 @@ def repos_for_topics(repos_by_topic, topics):
 
 
 class MetaRepoSyncer:
-    def __init__(self, clone_dir, name, submodules, author):
+    def __init__(self, clone_dir, name, submodules, author, sync_depth):
         self.clone_dir = clone_dir
         self.name = name
         self.submodules = submodules
         self.author = author
+        self.sync_depth = sync_depth
```

Comment:
Do you mind changing this to just depth? The sync prefix inside the sync class is a bit redundant.